libs/fixture-generators/src/generate-election/generate_election.ts

```diff
@@ -0,0 +1,184 @@
+import {
+  assertDefined,
+  DateWithoutTime,
+  mergeObjects,
+  range,
+} from '@votingworks/basics';
+import {
+  AnyContest,
+  BallotPaperSize,
+  BallotStyle,
+  District,
+  DistrictId,
+  Election,
+  ElectionId,
+  ElectionSchema,
+  Party,
+  PartyId,
+  Precinct,
+  safeParse,
+} from '@votingworks/types';
+import { customAlphabet } from 'nanoid';
+import { defaultConfig, GenerateElectionConfig } from './config';
+import { ballotMeasureText, seal, words } from './source_text';
+
+const generateId = customAlphabet('0123456789abcdefghijklmnopqrstuvwxyz', 12);
+
+function chooseRandom<T>(array: T[]): T {
```

Comment:
some of these util functions are probably also useful for other fixture generators, maybe there should be a general utils file they all share